signature/policy_eval_sigstore_test.go

```diff
@@ -1,7 +1,10 @@
-// Policy evaluation for prCosignSigned.
+//go:build cgo || containers_image_openpgp
```

Comment:
This is the other signing implementation (“sigstore”) and needs to work, so it needs to be tested, even if “simple signing” is disabled.